src/app/public/modules/tokens/token.component.ts

```diff
@@ -19,39 +19,36 @@ import {
 })
 export class SkyTokenComponent {
   @Input()
+  public get disabled(): boolean {
+    return !!this._disabled;
+  }
   public set disabled(value: boolean) {
     this._disabled = value;
   }
 
-  public get disabled(): boolean {
-    return (this._disabled === true);
-  }
-
   @Input()
+  public get ariaLabel(): string {
```

Comment:
Does putting the `@Input()` above the getter (as opposed to the setter) cause any problems here?

Reply:
Nah, it's functionally the same.